src/contract/assembled_transaction.ts

```diff
@@ -412,14 +446,16 @@ export class AssembledTransaction<T> {
   }
 
   simulate = async (): Promise<this> => {
-    if (!this.raw) {
-      throw new Error(
-        "Transaction has not yet been assembled; " +
-        "call `AssembledTransaction.build` first.",
-      );
-    }
+    if(!this.built){
```

Comment:
nit
```suggestion
    if (!this.built) {
```
(we'll get auto-fmt down eventually :laughing:)

Reply:
One day I'll remember to run yarn fmt :facepalm: